guides/acronyms.md

```diff
@@ -16,6 +16,8 @@
 
 *[PM]: Project Manager
 
+*[SEM]: Senior Engineering Manager
+
```

Comment:
Should "SFIA" be added to this? Apologies if it's defined elsewhere but I'm seeing it in these pages but not sure what it means? 🤯

Reply:
Anything like that should be added yeah. Can put it in a different branch though. 

https://sfia-online.org/en/sfia-8/sfia-8

Reply:
Sound, created issue #66 for this.